README.md

```diff
@@ -18,8 +18,13 @@ Build process uses s2i to package docker image.
 s2i build . quay.io/3scale/s2i-openresty-centos7:1.11.2.5-1-rover2  cors-proxy-app
 ```
 
+Run the container:
+```shell
+docker run --rm --name corsproxy -p 8080:8080 cors-proxy-app
```

Comment:
This will should be updated in https://github.com/3scale/cors-proxy/pull/32 IMO.

Reply:
I can rebase #32 after this one is merged.